src/game/Game.java

```diff
@@ -39,43 +59,66 @@ public void setP2Name(String p2Name) {
 		this.p2Name = p2Name;
 	}
 
-	String p1Name = "Player1";
-	String p2Name = "Player2";
-	// 0 is nobody, 1 is player1, 2 is player2
-	private int winner = 0;
-	private int numMoves = 4;
-	private int playerTurn = 1;
-	private boolean isPushPull;
+	public int getTurnNumber() {
+		return turnNumber;
+	}
 
-	public Game(BoardState b) {
-		currentBoard = b;
+	public void setTurnNumber(int turnNumber) {
+		this.turnNumber = turnNumber;
 	}
 
-	/**
-	 * Creates a board with a default starting layout
-	 */
-	public Game() {
-		currentBoard = new BoardState(
-				new char[][] { { 'K', 'D', 'H', 'C', 'E', 'H', 'D', 'K' }, { 'R', 'R', 'R', 'R', 'R', 'R', 'R', 'R' },
-						{ ' ', ' ', ' ', ' ', ' ', ' ', ' ', ' ' }, { ' ', ' ', ' ', ' ', ' ', ' ', ' ', ' ' },
-						{ ' ', ' ', ' ', ' ', ' ', ' ', ' ', ' ' }, { ' ', ' ', ' ', ' ', ' ', ' ', ' ', ' ' },
-						{ 'r', 'r', 'r', 'r', 'r', 'r', 'r', 'r' }, { 'k', 'd', 'h', 'c', 'e', 'h', 'd', 'k' }, },
-				0);
+	public void incrementTurn() {
+		this.turnNumber++;
 	}
 
-	/**
-	 * 
-	 * @param x
-	 * @param y
-	 * @return
-	 */
-	public Piece getSpace(int row, int column) {
-		if (row < 0 || row > 7 || column < 0 || column > 7)
-			return null;
-		if (currentBoard.getBoardArray()[row][column] == ' ')
-			return null;
-		return new Piece(currentBoard.getBoardArray()[row][column]);
+	public int getTurnCounter() {
+		return this.turnCounter;
+	}
+
+	public String getP1Name() {
+		return this.p1Name;
+	}
+
+	public String getP2Name() {
+		return this.p2Name;
+	}
+
+	public void setWinner(int winner) {
+		this.winner = winner;
+	}
 
+	public int getNumMoves() {
+		return numMoves;
+	}
+
+	public int getTurnTimer() {
+		return moveTimer;
+	}
+
+	public int getWinner() {
+		return winner;
+	}
+
+	public int getPlayerTurn() {
+		return playerTurn;
+	}
+
+	public void setPlayerTurn(int playerTurn) {
+		this.playerTurn = playerTurn;
+	}
+
+	@Deprecated
+	public AbstractPiece getSpace(int row, int column) {
```

Comment:
deprecated, preferred method is to get the current board and ask it for a piece at a coordinate